tools/swiftc_stub/main.swift

```diff
@@ -176,7 +176,8 @@ error: Failed to parse DEVELOPER_DIR from '-sdk'. Using /usr/bin/swiftc.
 // MARK: - Main
 
 let args = CommandLine.arguments
-if args.count == 2, args.last == "-v" {
+// Xcode 16.0 Beta 3 began using "--version" over "-v". Support both.
+if args.count == 2, args.last == "-v" || args.last == "--version" {
```

Comment:
Can you put the newer one first since it will become the main check going forward?